scripts/install-worker.sh

```diff
@@ -59,7 +59,6 @@ sudo yum install -y \
   aws-cfn-bootstrap \
   chrony \
   conntrack \
-  curl \
```

Comment:
I think `curl-minimal` is what we want on AL2023 -- that isn't on AL2 I assume?

Reply:
Correct. `curl-minimal` is only on AL2023.

Reply:
ack